src/components/SideTray/index.js

```diff
@@ -10,8 +10,10 @@ import style from './style.css';
 const SideTray = ({
   children,
   className,
+  closeOnOutsideClick,
   direction,
   height,
+  marginTop,
```

Comment:
I see a couple of issues with this approach.
1. `marginTop` implies that there should be configurability on other margins, and, since it's just a CSS style override, I'd rather see us spread a `styles` prop here (or something slightly more specific like `wrapperStyles`/`backdropStyles`).
2. Since we're setting a height to `100vh` by default (for both backdrop and content, I think), the margin will technically be pushing content off the bottom of the page.

I think, ideally, we'd put both backdrop and side tray content in a single wrapper div, add a `wrapperStyles` there, that applies `height: 100vh; width: 100vw; box-sizing: border-box; position: fixed;` by default. We'd have side tray and backdrop use relative positioning and `height: 100%;` as their default. 

Then, for this use case, set the `wrapperStyle` to `paddingTop: 40px;` and you get the child sizing for free.

Reply:
Thank you @pixelbandito ! Super helpful